packages/grafana-schema/src/raw/composable/logsnew/panelcfg/x/LogsNewPanelCfg_types.gen.ts

```diff
@@ -13,6 +13,7 @@ import * as common from '@grafana/schema';
 export const pluginVersion = "12.1.0-pre";
 
 export interface Options {
+  controlsStorageKey?: string;
```

Comment:
Is this saving the details controls in local storage?

Reply:
Correct. Can be passed from apps like Drilldown: https://github.com/grafana/logs-drilldown/blob/main/src/Components/ServiceScene/LogsPanelScene.tsx#L342